arcon_state/Cargo.toml

```diff
@@ -12,7 +12,7 @@ State management features for Arcon
 """
 
 [features]
-default = ["sled", "sled_checkpoints"]
+default = ["sled", "sled_checkpoints", "metrics"]
```

Comment:
metrics should not be a default feature for ``arcon_state``

We need to introduce a new ``state_metrics`` feature in the core Cargo.toml